### PR TITLE
Replace Blueprint.js elements with custom elements

### PR DESCRIPTION
**Pull request by Irtazaraza**
# Why is this PR needed?
closes: #4441 

# What does the PR do?
Replaces blueprint.js elements with custom elements

# Does it break backwards compatibility?
No

# List of future improvements not on this PR
- Replace blueprint tabs with custom tabs
- Replace blueprint loading button with custom button

### CODE REVIEW

File: workbase/src/renderer/components/shared/KeyspacesHandler.vue [hunk 1]
```diff
@@ -77,55 +88,55 @@
...
-      this.renderButton();
+    currentKeyspace(ks) {
+      if (ks !== null) {
+        if (ks.length > 15) { // truncate long keyspace names
+          this.btnText = `${ks.substring(0, 15)}...`;
+        } else {
+          this.btnText = ks;
+        }
+      } else {
+        this.btnText = 'keyspace';
```
Comment (marco-scoppetta) [inline workbase/src/renderer/components/shared/KeyspacesHandler.vue]: I guess the checks and string manipulations belong to filters

https://vuejs.org/v2/guide/filters.html

File: workbase/src/renderer/components/shared/KeyspacesHandler.vue [hunk 2]
```diff
@@ -106,12 +106,22 @@ export default {
...
+    truncate(ks) {
+      if (ks !== null) {
+        if (ks.length > 15) { // truncate long keyspace names
+          return `${ks.substring(0, 15)}...`;
+        }
+        return ks;
+      }
+      return 'keyspace';
+    },
+  },
```
Comment (marco-scoppetta) [inline workbase/src/renderer/components/shared/KeyspacesHandler.vue]: I think we can make it a bit shorter with:
```
if(!ks) return 'keyspace';

if (ks.length > 15) return `${ks.substring(0,15)}...`;

return ks
```

File: workbase/src/renderer/components/shared/KeyspacesHandler.vue [hunk 3]
```diff
@@ -122,17 +132,17 @@ export default {
...
+    //   if (ks !== null) {
+    //     if (ks.length > 15) { // truncate long keyspace names
+    //       this.btnText = `${ks.substring(0, 15)}...`;
+    //     } else {
+    //       this.btnText = ks;
+    //     }
+    //   } else {
+    //     this.btnText = 'keyspace';
+    //   }
+    // },
```
Comment (marco-scoppetta) [inline workbase/src/renderer/components/shared/KeyspacesHandler.vue]: 💃

Comment (Irtazaraza) [reply workbase/src/renderer/components/shared/KeyspacesHandler.vue]: 🙈

**Pull request merged**